x-pack/plugins/uptime/public/components/fleet_package/custom_fields.tsx

```diff
@@ -122,6 +124,32 @@ export const CustomFields = memo<Props>(
                   />
                 </EuiFormRow>
               )}
+              <EuiSpacer size="s" />
+              {isBrowser && (
+                <EuiCallOut
+                  title={
+                    <FormattedMessage
+                      id="xpack.uptime.createPackagePolicy.stepConfigure.monitorIntegrationSettingsSection.monitorType.browser.warning.description"
+                      defaultMessage={
+                        'Browser monitoring requires using the "complete" variant of the elastic–agent docker container. Other distributions of elastic that agent will not work correctly with it because they lack the required browser and other binaries. For more information see our {link}'
+                      }
+                      values={{
+                        link: (
+                          <EuiLink target="_blank" href="" external>
```

Comment:
This link still needs to be provided.

Reply:
For now I'd say `https://www.elastic.co/guide/en/observability/current/synthetic-monitoring.html` is fine. I've opened https://github.com/elastic/observability-docs/issues/989 to track adding more docs